frontend/prisma/schema.prisma

```diff
@@ -15,16 +15,46 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model User {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  email     String?  @unique
-  image     String?
-  role      Role     @default(USER)
+model Clubs {
+  cid         String         @id @default(uuid())
+  email       String         @unique
+  password    String
+  info        ClubInfo[]
```

Comment:
do we want a club to have multiple ClubInfo?

Reply:
No, I don't think so

Reply:
Do we want to require a club to have club info?